convert2rhel/unit_tests/utils_test.py

```diff
@@ -207,11 +207,19 @@ def test_download_pkg_success_with_all_params(self):
 
     @unit_tests.mock(system_info, "version", namedtuple("Version", ["major", "minor"])(7, 0))
     @unit_tests.mock(utils, "run_cmd_in_pty", RunSubprocessMocked(ret_code=1))
-    def test_download_pkg_failed_download(self):
+    @unit_tests.mock(os, "environ", {"CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK": "1"})
+    def test_download_pkg_failed_download_overrridden(self):
         path = utils.download_pkg("kernel")
 
         self.assertEqual(path, None)
 
+    @unit_tests.mock(system_info, "version", namedtuple("Version", ["major", "minor"])(7, 0))
+    @unit_tests.mock(utils, "run_cmd_in_pty", RunSubprocessMocked(ret_code=1))
+    @unit_tests.mock(os, "environ", {})
+    def test_download_pkg_failed_download_exit(self):
+
+        self.assertRaises(SystemExit, utils.download_pkg, "kernel")
```

Comment:
```suggestion
        self.assertRaises(SystemExit, utils.download_pkg, "kernel")
```

Reply:
I have gotten rid of the extra line in this comment